modules/oe_content_event/config/install/field.field.node.oe_event.body.yml

```diff
@@ -0,0 +1,21 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.body
+    - node.type.oe_event
+  module:
+    - text
+id: node.oe_event.body
+field_name: body
+entity_type: node
+bundle: oe_event
+label: 'Full text'
+description: ''
+required: false
```

Comment:
should be required

Reply:
Done through constraint. By documentation this field is required only if any fields within Description fields group filled in.

Reply:
As discussed with @nagyad this is missing a field in the check.

Reply:
I have added Description summary and Description title in composite section of Description.

Reply:
fixed